pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authorization/AuthorizationProvider.java

```diff
@@ -183,6 +183,16 @@ default void initialize(ServiceConfiguration conf, PulsarResources pulsarResourc
     CompletableFuture<Boolean> allowSinkOpsAsync(NamespaceName namespaceName, String role,
                                                  AuthenticationDataSource authenticationData);
 
+    /**
+     * Allow package operations with in this namespace
+     * @param namespaceName The namespace that the package operations can be executed in
+     * @param role The role to check
+     * @param authenticationData authentication data related to the role
+     * @return a boolean to determine whether authorized or not
+     */
+    CompletableFuture<Boolean> allowPackageOpsAsync(NamespaceName namespaceName, String role,
```

Comment:
@yuruguo - can you explain why we need this method in the interface? It is only used internally by the `PulsarAuthorizationProvider` and `MultiRolesTokenAuthorizationProvider`, so I don' think we need to add it to the public interface.

Reply:
It is clearer to wrap the `packages` operation into proprietary methods, which is the same as `allowConsumeOpsAsync`, although `allowConsumeOpsAsync` also only be used in internally by the `PulsarAuthorizationProvider` and `MultiRolesTokenAuthorizationProvider`. I have no objection if you insist on your point of view :)
https://github.com/apache/pulsar/blob/2e963a30a112a715f8a97682d671a0e8eab6cb81/pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authorization/PulsarAuthorizationProvider.java#L228-L231

Reply:
Thank you for pointing out that new method. I think we should remove that one too, so I just submitted a PR to do so: https://github.com/apache/pulsar/pull/13133.

Note that I am fine with custom methods to wrap the `packages` operation. However, I think we need to be careful about how we expand our public API.

Reply:
Okay! I have remove `allowPackageOpsAsync` method and directly use `allowTheSpecifiedActionOpsAsync`.